docker/postgrest.conf

```diff
@@ -9,6 +9,7 @@ server-port = "$(PGRST_SERVER_PORT)"
 server-proxy-uri = "$(PGRST_SERVER_PROXY_URI)"
 jwt-secret = "$(PGRST_JWT_SECRET)"
 secret-is-base64 = "$(PGRST_SECRET_IS_BASE64)"
+jwt-aud = "$(PGRST_JWT_AUD")
```

Comment:
Typo here, it should be `jwt-aud = "$(PGRST_JWT_AUD)"`

Reply:
Thank you! Fixed.